elastic/integrations
# [synthetics] Update synthetics minor for 8.11 changes

## Proposed commit message



Update Synthetics integration minor version for changes starting from Kibana v8.11 .

## Checklist


## How to test this PR locally

1. Build integration locally.
2. Install remotely to a cluster with `elastic-package install`.
3. Check installed integration version is `1.1.0`.

## Related issues


- Relates #7755.
